composer.json

```diff
@@ -1,14 +1,14 @@
 {
-    "name": "bitpay/php-client",
-    "description": "PHP Library to work with the new cryptographically secure BitPay API",
```

Comment:
`PHP library to work with BTCPay Server` the API is now diverging from Bitpay, so I think we can stop mentioning them.